src/shared-content/upgrade/upgrade-find-and-backup-master-key.md

```diff
@@ -0,0 +1,12 @@
+### Finding and backing up your Master Key
+
+When connecting an existing Octopus database to a "new" Octopus instance, either due to migration or for testing, you will need the Master Key to gain access to the database during the Octopus instance setup process.
+
+To obtain your existing Master Key from your source Octopus instance, you can simply open Octopus Manager then select "View master key" as shown below:
+
+
+:::figure
+![](/docs/upgrade/images/view-master-key.png)
+:::
+
+Alternatively, you may also use the `show-master-key` command via the `Octopus.Server.exe` command-line tool. [You can find more information on this here](https://octopus.com/docs/octopus-rest-api/octopus.server.exe-command-line/show-master-key).
```

Comment:
You might want to adjust these settings to account for linux containers, so making it more platform-agnostic.

Reply:
See commit [40d5dd0](https://github.com/OctopusDeploy/docs/pull/2019/commits/40d5dd0f5e5df9c7fa70d6257517d9e959a9e882)